server/server.go

```diff
@@ -691,19 +511,43 @@ func (s *Server) spawn(l net.Listener, c net.Conn) {
 		tconn.Close()
 		return
 	}
-	handle := client.SpawnConn(conn, s.config.maxConnPendingRequests)
-	s.listeners[l][handle] = struct{}{}
+
+	s.listeners[l][tconn] = struct{}{}
 	s.mtx.Unlock()
-	log.Debugf("%s: spawned", connStr)
+	log.Debugf("%s: serving", connStr)
+	handler := &handler{
+		name:        connStr,
+		outstanding: s.config.maxConnPendingRequests,
```

Comment:
Shouldn't this be zero? We start out with 0 outstanding requests, when we begin to handle a request, [we increase `outstanding`](https://github.com/cloudflare/gokeyless/pull/292/files#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4R177), when we are done handling it, [we decrease outstanding](https://github.com/cloudflare/gokeyless/pull/292/files#diff-78f42ba40d0f10b08c73b7e6bb8376f398e249c963cf549e89591d6b6826b9a4R535)? Or is that not the case?

Reply:
I think I should rename this variable to "tickets" or something.

Reply:
Or remainingCapacity or capacity

Reply:
I've replaced it with a channel